app/components/App/App.jsx

```diff
@@ -1,39 +1,28 @@
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { bindActions } from '../../actions';
-import Header from '../Header';
-
-if (process.env.BROWSER) {
-  require('../../stylesheets/defaults/content.scss');
-}
-
-function renderChildren (props) {
-  return React.Children.map(props.children, function(child) {
-    return React.cloneElement(child, {
-      actions: props.actions,
-      state: props.state
-    } );
-  });
-}
+import MainWrapper from '../MainWrapper';
 
 function App (props) {
   return (
-    <main>
-      <Header />
-      <div className='content'>
-        { renderChildren(props) }
-      </div>
-    </main>
+    <MainWrapper
+      actions={ props.actions }
+      children={ props.children }
+      history={ props.history }
+      location={ props.location }
+      state={ props.state } />
   );
 }
 
 App.propTypes = {
   actions: PropTypes.object.isRequired,
-  children: PropTypes.element.isRequired,
+  children: PropTypes.object.isRequired,
+  history: PropTypes.object.isRequired,
+  location: PropTypes.object.isRequired,
   state: PropTypes.object.isRequired
 };
 
 export default connect(
-  function mapStateToProps(state) { return { state: state }; },
+  function mapStateToProps(state) { return { state }; },
```

Comment:
this seems funny to me. maybe I am wrong. Does it need to be `return state;` or `return {...state}`??